muesli/web/templates/index.pt

```diff
@@ -1,24 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
-<metal:main metal:use-macro="templates('Fragments/main.pt').macros['main']">
-  <metal:main metal:fill-slot="content">
-<h2 class="display-4">Willkommen bei MÜSLI!</h2>
-
-    <p class="lead">
-    MÜSLI verwaltet viele Übungsgruppen und Vorlesungen an der Fakultät für Mathematik und Informatik der Universität Heidelberg.
-    </p>
-
-    <p>
-    Um sich zu Übungsgruppen anmelden zu können, müssen Sie sich zunächst <a tal:attributes="href request.route_path('user_register')">bei MÜSLI registrieren</a>.
-    Sind Sie bereits registriert, müssen Sie sich <a tal:attributes="href request.route_path('user_login')">anmelden</a>.
-    Danach wählen Sie Ihre Vorlesung aus der <a tal:attributes="href request.route_path('lecture_list')">Liste der mit MÜSLI verwalteten Vorlesungen</a> aus und folgen den Anweisungen dort.
-    </p>
-
-    <p>
-    Diese Version von MÜSLI wurde in den Semesterferien zum Wintersemester 2011/12 neu aufgesetzt.
-    </p>
-
-    <p>
-    Das MÜSLI-Team.
-    </p>
-  </metal:main>
-</metal:main>
```

Comment:
I would like to hear the rationale behind removing the start page!
To me it felt really awkward not having it and being instantly redirected to the login page.

Changing the behaviour in a way that you i.e. get redirected to the `/start` if you are logged in would make sense, but removing it entirely doesn't make sense to me ATM; I am excited for to hear your reasoning!

Reply:
I removed the start page due to the one use case of MÜSLI which I think is most prevalent for the majority of users (at least I know it's the case for my friends and me): A student visits the site and immediately clicks on "log in" because they want to have a look at their tutorials/points/...

My intentions were to save them one click.

If someone without an account is interested more in MÜSLI, I'd rather favor the creation of another "about" page with more details someone can reach after doing one click.